docs/_templates/changelog/v3.0.0.md

```diff
@@ -26,10 +26,12 @@
 
 ### `🧱 bs.block`
 
+- <abbr title="Breaking Changes">⚠️</abbr> **[#375](https://github.com/mcbookshelf/bookshelf/pull/375)** - Changed `get_block` and `get_type` to only return existing data. Empty strings for state or NBT are no longer possible; if absent, the field is omitted instead.
 - <abbr title="New Features">✨</abbr> **[#349](https://github.com/mcbookshelf/bookshelf/issues/349)** - Added the `on_finished` command argument to block fill operations. If specified the provided string is run as a command once the block fill operation is finished.
 - <abbr title="New Features">✨</abbr> **[#279](https://github.com/mcbookshelf/bookshelf/issues/279)** - Added the `#bs.block:play_block_sound` function for playing block sounds.
 - <abbr title="New Features">✨</abbr> **[#287](https://github.com/mcbookshelf/bookshelf/issues/287)** - Updated `replace_type` to return whether a type was found and replaced.
 - <abbr title="Bug Fix">🐛</abbr> **[#320](https://github.com/mcbookshelf/bookshelf/issues/320)** - Fixed functions that were unusable outside the Overworld.
+- <abbr title="Enhancements">⚡</abbr> **[#375](https://github.com/mcbookshelf/bookshelf/pull/375)** - Updated `emit_block_particle` and `play_block_sound` to support default values. Added `mode` and `viewers` parameters to `emit_block_particle`.
```

Comment:
I think we can remove the note concerning "play_block_sound" since… This feature has been added in this version, so this is not a change.